lib/modules/manager/bundler/artifacts.ts

```diff
@@ -28,6 +32,26 @@ import {
 
 const hostConfigVariablePrefix = 'BUNDLE_';
 
+export function buildArgs(config: UpdateArtifactsConfig): string[] {
+  const args: string[] = [];
+  // --major is the default and does not need to be handled separately.
+  switch (config.updateType) {
+    case 'patch':
+      args.push('--patch', '--strict');
```

Comment:
Should we change the docs in this PR as well? Our docs mention some limitations for our Ruby bundler support: [^quote]

> ## Warnings
> 
> Renovate doesn't update dependencies without a version constraint.
> Example: `gem 'some-gem', '~> 1.2.3'` will update `some-gem` if a new version matching the constraint is available, but `gem 'some-gem'` won't.
> If you always want to have the latest available version, consider specifying `gem 'some-gem', '> 0'`.

[^quote]: https://docs.renovatebot.com/ruby/#warnings

Reply:
I think this warning still applies?

Reply:
I think this warning was already obsolete before this pull request.

In a private project where we use Renovate, we have almost no versions in our Gemfile, only stuff like `gem "redis"`. Renovate still creates update PRs for those gems (we have `"rangeStrategy": "update-lockfile"` specified).

Reply:
I found that at least with `"rangeStrategy": "bump"` it would not update gems that do not have a constraint.
But it will (and has before this PR) with `"rangeStrategy": "update-lockfile"`. With this PR this now even for works non-major updates.

Maybe the warning could be updated to recommend `"rangeStrategy": "update-lockfile"` for those cases?

Reply:
I updated the `Warnings` in the ruby usage section according to this discussion. Let me know what you think, and if it is understandable 🙂